### ISSUE TITLE
Using a button as the accordion__label causes styling to break

### ISSUE THREAD

**Issue by rbrtsmith** [sky-uk/toolkit#346]
## Expected Behaviour

The styling for this component should be consistent whether using a button or a label

## Current Behaviour

When using a button for the `accordion__label` the button collapses due to buttons not filling their container even when `display: block` is applied, buttons are also loaded with some default properties too such as `background-color` and `text-align`.
<img width="874" alt="screen shot 2017-12-18 at 14 09 34" src="https://user-images.githubusercontent.com/4982001/34110383-6cfb74b6-e3fe-11e7-97fe-0c2e9031f0cf.png">

## Attempted Fixes

Added the following classes to `accordion__label`
```
width: 100%;
border-width: 0;
text-align: inherit;
background-color: transparent;
```
<img width="878" alt="screen shot 2017-12-18 at 14 05 52" src="https://user-images.githubusercontent.com/4982001/34110410-7dbe5f20-e3fe-11e7-8865-4ff153386298.png">
Fix applied to first item in the accordion example here.

## Steps to Reproduce (for bugs)

Provide a link to a live example, or an unambiguous set of steps to
reproduce this bug. Include code to reproduce, if relevant

1.  Open Toolkit React
2. Update the accordionSection swapping out the anchor for a button
3. Run in preview mode.

## Context

This change is required to satisfy an accessibility linting error that has surfaced in TKR as part of [this pr](https://github.com/sky-uk/toolkit-react/pull/311)

## Your Environment

* Version used: `8.4.1`
* Environment name and version (e.g. Chrome 39, node.js 5.4): `Node 7.10.1`, `Chrome 62`
* Operating System and version (desktop or mobile): OSX Sierra.
